cmd/oci/download.go

```diff
@@ -168,6 +170,31 @@ Examples:
 			}
 		}
 
+		if opts.uncompressGzFiles {
+			var extractErrors []error
+
+			gzFilesFromOciArtifacts, err := ociController.GetGzFilesFromDir(opts.artifactsOutput)
+			if err != nil {
+				return fmt.Errorf("failed to extract gz files from artifacts folder: %w", err)
+			}
+
+			for _, file := range gzFilesFromOciArtifacts {
+				err := ociController.ExtractGzFile(file.FilePath, file.DirPath)
+				if err != nil {
+					extractErrors = append(extractErrors, err)
+					log.Printf("warn: file %s was not extracted successfully: %s", file.FilePath, err.Error())
+				}
+
+				for _, err := range extractErrors {
+					log.Printf(" - %v\n", err)
+				}
```

Comment:
should this `for` loop rather be **out** of the current `for` loop?

or **do we even need** to collect and print out `extractErrors`? the error is already logged [here](https://github.com/konflux-ci/qe-tools/pull/146/files#diff-90ed32292428dbdff9e80fb879ed23a54633f2afb744e302d9cf7af9ac0cfb78R185)